src/components/SEARCHBAR/SearchBar.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState} from 'react';
+import style from './../HEADER/Header.module.css';
```

Comment:
import headerStyles from './../HEADER/Header.module.css';

Reply:
Should I change it everywhere or only in searchBar?